preprocessors/multistage-textbook-diagram.schema.json

```diff
@@ -0,0 +1,149 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/mutistage-textbook-diagram.schema.json",
+    "type": "object",
+    "title": "Multistage Textbook Diagram Data",
+    "description": "JSON containing pixel coordinates of different stages in a mutistage textbook diagram and the links between them",
```

Comment:
Why is this only for textbooks?

Reply:
Fair! I think this can be called "Multistage Diagram Data" instead.

The reason this is being defined separately from semantic segmentation is because there are multiple different sections in the graphics (e.g. cell cycle, human evolution stages, life cycle diagrams) for which this is intended. Knowing how they are linked could help better define individual/ detailed layers for each of them and also represent the links without introducing unnecessary clutter especially if decorative visual arrows etc. are being used in the source graphic.